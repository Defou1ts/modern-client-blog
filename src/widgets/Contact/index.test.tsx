import React from 'react';

import { render, screen } from '@testing-library/react';

import renderer from 'react-test-renderer';

import { Contact } from './index';

describe('Contact', () => {
	it('renders a  Contact', () => {
		render(<Contact />);
		const text = screen.getByText('contact.subtitle');
		expect(text).toBeInTheDocument();
		expect(text).toBeVisible();
	});

	it('Contact snapshot', () => {
		const component = renderer.create(<Contact />);

		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
