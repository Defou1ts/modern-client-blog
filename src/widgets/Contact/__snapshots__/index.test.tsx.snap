// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Contact snapshot 1`] = `
<div
  className="wrapper"
>
  <div
    className="wrapper"
  >
    <h6
      className="h h6 contact"
    >
      contact.title
    </h6>
    <h1
      className="h h1 conversation"
    >
      contact.subtitle
    </h1>
    <p
      className="p medium description"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim.
    </p>
  </div>
  <div
    className="wrapper"
  >
    <div
      className="item"
    >
      <p
        className="p small itemTitle"
      >
        contact.working.title-days
      </p>
      <div
        className="line"
      />
      <h5
        className="h h5 info"
      >
        contact.working.days
         
        <br />
         
        contact.working.hours
         
      </h5>
      <p
        className="p medium support"
      >
        contact.working.availability
      </p>
    </div>
    <div
      className="item"
    >
      <p
        className="p small itemTitle"
      >
        contact.working.title-contact
      </p>
      <div
        className="line"
      />
      <h5
        className="h h5 info"
      >
        020 7993 2905
      </h5>
      <p
        className="p medium support"
      >
        hello@finsweet.com
      </p>
    </div>
  </div>
</div>
`;
